plugin/handlergen.go

```diff
@@ -270,29 +259,8 @@ func (p *OrmPlugin) generatePatchHandler(message *generator.Descriptor) {
 	p.P(`}`)
 	p.P(`}`)
 
-	// Convert pbObj back to ormObj to trigger any logic that was
-	// written for BeforeToORM/AfterToORM and perform db.Save call.
-	p.P(`ormObj, err = pbObj.ToORM(ctx)`)
-	p.P(`if err != nil {`)
-	p.P(`return nil, err`)
-	p.P(`}`)
-
-	if isMultiAccount {
-		p.P(`db = db.Where(&`, typeName, `ORM{AccountID: accountID})`)
-	}
-
-	p.P(`if err = db.Save(&ormObj).Error; err != nil {`)
-	p.P(`return nil, err`)
-	p.P(`}`)
-
-	// convert ormObj to pbObj again (sic!) to trigger any logic that was
-	// written for AfterToPB/BeforeToPB.
-	p.P(`pbObj, err = ormObj.ToPB(ctx)`)
-	p.P(`if err != nil {`)
-	p.P(`return nil, err`)
-	p.P(`}`)
+	p.P(`return DefaultStrictUpdate`, typeName, `(ctx, &pbObj, db)`)
```

Comment:
Definitely cleaner (and also fixes the issue of removing associations in the ApplyFieldMask method). Have you tested it in practice to make sure it works correctly like this?

Reply:
build passes, and only thing that has changed is apply field mask is now not removes associations (the're all are removed in StrictUpdate)